docker/run.sh

```diff
@@ -17,6 +17,7 @@ mysql.exe = /usr/bin/mysql
 node.exe = /usr/bin/node
 publisher.js = /app/BiblePublisher/publish/Publisher.js
 sofria_client.js =  /app/sofria-cli/sofria_mediaId.js
```

Comment:
(see general note for context)
change sofria_client.js to /app/sofria-cli/biblebrain_uploader.js
remove gqlquery_verses_by_usx.js

Reply:
@bradflood , it has been changed and for now on, it will invoke a one call to generate the JSON formatted files and to populate the database.